starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,42 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+h1{
+  color: white;
+  background-image: url('../images/soup.jpg');
+  height: 500px;
+  width: 80%;
+  background-size: cover;
+  text-align: center;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  line-height: 450px;
+  font-size: 75px;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+hr{
+  width: 75%;
+}
+
```

Comment:
too many repeating styles. A good work around is to wrap the tags that are together in a div and then adjust the margin and/or padding of the div